### PR TITLE
Updates to override class members

### PR DESCRIPTION
**Pull request by elsh**

### CODE REVIEW

File: Sources/MockoloFramework/Models/VariableModel.swift [hunk 1]
```diff
@@ -68,21 +68,29 @@ final class VariableModel: Model {
         self.filePath = filepath
     }
     
+    private func isGenerated(_ name: String) -> Bool {
+        return name.hasPrefix(.underlyingVarPrefix) || name.hasSuffix(.setCallCountSuffix) || name.hasSuffix(.callCountSuffix) || name.hasSuffix(.subjectSuffix) || name.hasSuffix("SubjectKind") || name.hasSuffix(.handlerSuffix)
```
Comment (NicoHinderling) [inline Sources/MockoloFramework/Models/VariableModel.swift]: how come "SubjectKind" isn't its own enum case?

Comment (elsh) [reply Sources/MockoloFramework/Models/VariableModel.swift]: ah yes, will update

File: Sources/MockoloFramework/Templates/ClassTemplate.swift [hunk 2]
```diff
@@ -94,22 +93,6 @@ private func extraInitsIfNeeded(_ entities: [(String, Model)],
...
-                    if !matchingParams.isEmpty {
-                        matchingParamsFound = true
-                        break
-                    }
-                }
-            }
-            
-            if matchingParamsFound || extraInitParamCandidates.isEmpty {
-                needParamedInit = false
-            }
```
Comment (NicoHinderling) [inline Sources/MockoloFramework/Templates/ClassTemplate.swift]: How come we don't need any of this anymore? Admittedly I don't have context on the role that `needParamedInit` plays, so I'm assuming it's related to that

Comment (elsh) [reply Sources/MockoloFramework/Templates/ClassTemplate.swift]: extra init's are still generated for protocol mocks but not for class mocks.  this change handles that.

**Pull request merged**